yarn-project/cli-wallet/src/cmds/index.ts

```diff
@@ -0,0 +1,161 @@
+    )
+    .summary('Creates an aztec account that can be used for sending transactions.')
+    .option(
+      '--skip-initialization',
+      'Skip initializing the account contract. Useful for publicly deploying an existing account.',
+    )
+    .option('--public-deploy', 'Publicly deploys the account and registers the class if needed.')
+    .addOption(createPrivateKeyOption('Private key for account. Uses random by default.', false))
+    .option('-a, --alias <string>', 'Alias for the account. Used for easy reference in the PXE.');
+
+  addOptions(createAccountCommand, FeeOpts.getOptions())
+    .option(
+      '--register-only',
+      'Just register the account on the PXE. Do not deploy or initialize the account contract.',
+    )
+    // `options.wait` is default true. Passing `--no-wait` will set it to false.
+    // https://github.com/tj/commander.js#other-option-types-negatable-boolean-and-booleanvalue
+    .option('--no-wait', 'Skip waiting for the contract to be deployed. Print the hash of deployment transaction')
+    .action(async (_options, command) => {
+      const { createAccount } = await import('../cmds/create_account.js');
+      const options = command.optsWithGlobals();
+      const { privateKey, wait, registerOnly, skipInitialization, publicDeploy, rpcUrl, alias } = options;
+      await createAccount(
+        rpcUrl,
+        privateKey,
+        alias,
+        registerOnly,
+        skipInitialization,
+        publicDeploy,
+        wait,
+        FeeOpts.fromCli(options, log),
+        debugLogger,
+        log,
+      );
+    });
+
+  const deployCommand = program
+    .command('deploy')
+    .description('Deploys a compiled Aztec.nr contract to Aztec.')
+    .argument(
+      '<artifact>',
+      "A compiled Aztec.nr contract's artifact in JSON format or name of a contract artifact exported by @aztec/noir-contracts.js",
+    )
+    .option('--init <string>', 'The contract initializer function to call', 'constructor')
+    .option('--no-init', 'Leave the contract uninitialized')
+    .option('--args <constructorArgs...>', 'Contract constructor arguments', [])
+    .option(
+      '-k, --public-key <string>',
+      'Optional encryption public key for this address. Set this value only if this contract is expected to receive private notes, which will be encrypted using this public key.',
+      parsePublicKey,
+    )
+    .option(
+      '-s, --salt <hex string>',
+      'Optional deployment salt as a hex string for generating the deployment address.',
+      parseFieldFromHexString,
+    )
+    .option('--universal', 'Do not mix the sender address into the deployment.')
+    .option(
+      '-a, --alias-or-address <string>',
+      'Alias or address of the account to deploy from. Incompatible with --private-key.',
+    )
```

Comment:
You can use `.conflicts` to make commander check this for you: https://github.com/tj/commander.js/blob/970ecae402b253de691e6a9066fea22f38fe7431/examples/options-conflicts.js#L8. This would help model the 'either alias or pk' pattern we have going on that I referenced before

Reply:
Done!